### PR TITLE
added 4 tokens : OLAS, sDAI, stETH and EURe - on gnosis chain

### PR DESCRIPTION
**Pull request by PoloX2021**
Added 4 tokens to gnosis_chain

### CODE REVIEW

**Comment by PoloX2021** [comment]
stETH : https://gnosisscan.io/address/0x3c037849a8ffcf19886e2f5b04f293b7847d0377
EURe : https://gnosisscan.io/address/0xcb444e90d8198415266c6a2724b7900fb12fc56e
sDAI : https://gnosisscan.io/address/0xaf204776c7245bf4147c2612bf6e5972ee483701
OLAS : https://gnosisscan.io/address/0xce11e14225575945b8e6dc0d4f2dd4c570f79d9f

File: models/prices/gnosis/prices_gnosis_tokens.sql [hunk 1]
```diff
@@ -31,5 +31,9 @@ FROM
     ('crv-curve-dao-token', 'gnosis', 'CRV', 0x712b3d230f3c1c19db860d80619288b1f0bdd0bd, 18),
     ('link-chainlink', 'gnosis', 'LINK', 0xe2e73a1c69ecf83f464efce6a5be353a37ca09b2, 18),
     ('sushi-sushi', 'gnosis', 'SUSHI', 0x2995d1317dcd4f0ab89f4ae60f3f020a4f17c7ce, 18),
-    ('wsteth-wrapped-liquid-staked-ether-20', 'gnosis', 'WSTETH', 0x6c76971f98945ae98dd7d4dfca8711ebea946ea6, 18)     
+    ('wsteth-wrapped-liquid-staked-ether-20', 'gnosis', 'WSTETH', 0x6c76971f98945ae98dd7d4dfca8711ebea946ea6, 18),
+    ('sdai-savings-dai', 'gnosis', 'SDAI',0xaf204776c7245bF4147c2612BF6e5972Ee483701,18),
+    ('eure-monerium-eur-emoney', 'gnosis', 'EURe', 0xcB444e90D8198415266c6a2724b7900fb12FC56E, 18),
+    ('olas-autonolas', 'gnosis', 'OLAS', 0xcE11e14225575945b8E6Dc0D4F2dD4C570f79d9f, 18),
+    ('steth-lido-staked-ether-on-xDAI', 'gnosis', 'stETH', 0xae7ab96520de3a18e5e111b5eaab095312d7fe84, 18)
```
Comment (Hosuke) [inline models/prices/gnosis/prices_gnosis_tokens.sql:38]: Hi @PoloX2021,
Do you think `steth-lido-staked-ether` works here? As `steth-lido-staked-ether-on-xDAI` is not a legal token-id.
```suggestion
    ('steth-lido-staked-ether', 'gnosis', 'stETH', 0xae7ab96520de3a18e5e111b5eaab095312d7fe84, 18)
```

error msg:
https://github.com/duneanalytics/spellbook/actions/runs/8751643700/job/24017683495?pr=5794#step:6:40

**Review by Hosuke — CHANGES_REQUESTED** [review]

File: models/prices/gnosis/prices_gnosis_tokens.sql (hunk 1 above)
Comment (PoloX2021) [reply models/prices/gnosis/prices_gnosis_tokens.sql]: It's done I changed the name to what you suggested. i also noticed that the address was the one for ethereum so I changed it to the one for gnosis. (I also double checked all the other ones ;) )

**Review by Hosuke — APPROVED** [review]
LGTM.✅
Thank you @PoloX2021

**Pull request merged**